app/services/timers/promise-timeout.service.ts

```diff
@@ -0,0 +1,17 @@
+const PromiseTimeout = (ms: number, promise: Promise < any > ): Promise < any > => {
+	// Create a promise that rejects in <ms> milliseconds
```

Comment:
why do we need that?

Reply:
if it's for server request we can define timeout on Axios...

Reply:
yes this is for timeout request, in case of timeout that i defined , it would throws an exception, please send me a link of what you mean with Axios, i know Axios from nestJS, but specifically i don't know this possibility.